content/python/concepts/built-in-functions/terms/complex/complex.md

```diff
@@ -22,11 +22,17 @@ Converts a given string into a complex number.
 complex(real, imaginary)
 ```
 
-## Example 1
+## Example 
```

Comment:
These edits are just modifying the existing example. This example can stay as-is and you need to add a new example, ideally an example that is different from the existing (demonstrates some other functionality, a special case, etc) as a codebyte.